File: ProAgil.Domain/Speaker.cs
using System.Collections.Generic;

namespace ProAgil.Domain
{
    public class Speaker
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string MiniCurriculum { get; set; }
        public string ImageUrl { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public List<SocialNetwork> SocialNetworks {get; set;}
        public List<SpeakerEvent> SpeakerEvents { get; set; }
    }
}

File: ProAgil-App/src/app/_models/Speaker.ts
import { SocialNetwork } from "./SocialNetwork";
import { Event } from "./Event";

export interface Speaker {
  id: number;
  interface: string;
  miniCurriculum: string;
  imageUrl : string;
  phone: string;
  email: string;
  socialNetworks: SocialNetwork[];
  speakerEvents: Event[];
}
